client/state/data-layer/wpcom/wordads/stats/index.js

```diff
@@ -0,0 +1,27 @@
+/** @format */
+
+/**
+ * Internal dependencies
+ */
+import { http } from 'state/data-layer/wpcom-http/actions';
+import { dispatchRequestEx } from 'state/data-layer/wpcom-http/utils';
+import { errorNotice } from 'state/notices/actions';
+import { WORDADS_STATS_REQUEST } from 'state/action-types';
+import { receiveStats } from 'state/wordads/stats/actions';
+
+export default {
+	[ WORDADS_STATS_REQUEST ]: [
+		dispatchRequestEx( {
+			fetch: action =>
+				http(
+					{
+						method: 'GET',
+						path: `/sites/${ action.siteId }/wordads/stats`,
```

Comment:
Is this data layer handler used anywhere? I see that in this PR you add both a `wpcom.undocumented` method and a data layer handler. That's suspicious -- there should be just one of them, not both. Especially in new code.